addons/binding/org.openhab.binding.robonect/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,254 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="robonect"
+                          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                          xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+                          xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+    
+    <thing-type id="mower">
+        <label>Mower</label>
+        <description>Mower robot connected via robonect module</description>
+        <channel-groups>
+            <channel-group id="mowerInfo" typeId="mowerInfoType"/>
+            <channel-group id="mowerStatus" typeId="mowerStatusType"/>
+            <channel-group id="timer" typeId="timerType"/>
+            <channel-group id="wlan" typeId="wlanType"/>
+            <channel-group id="job" typeId="jobGroupType"/>
+            <channel-group id="error" typeId="errorGroupType"/>
+            <channel-group id="version" typeId="robonectVersionType"/>
+        </channel-groups>
+        <config-description>
+            <parameter name="host" type="text" required="true">
+                <label>Host</label>
+                <description>Host name or network address of the Robonect module</description>
+            </parameter>
+            <parameter name="user" type="text" required="false">
+                <label>User</label>
+                <description>The user id if authentication has been enabled on the mower</description>
+            </parameter>
+            <parameter name="password" type="text" required="false">
+                <label>Password</label>
+                <description>The password if authentication has been enabled on the mower</description>
+            </parameter>
+            <parameter name="pollInterval" type="integer" required="false" unit="s">
+                <label>Polling interval.</label>
+                <description>The interval for the binding to poll the mowers status information. Defaults to 30
+                    seconds.
+                </description>
+            </parameter>
```

Comment:
cool, I was looking for that, but somehow I seem to be blind ;-) thx for mention